src/main/java/no/nav/fo/veilarbregistrering/arbeidssoker/Arbeidssokerperioder.java

```diff
@@ -38,18 +41,27 @@ public boolean dekkerHele(Periode forespurtPeriode) {
     }
 
     public Arbeidssokerperioder sorterOgPopulerTilDato() {
-        return new Arbeidssokerperioder(of(this.arbeidssokerperioder.stream()
-                .sorted(Comparator.comparing(e -> e.getPeriode().getFra())).collect(Collectors.toList()))
-                .map(populerTilDato).get());
+        return new Arbeidssokerperioder(
+                of(arbeidssokerperioder.stream()
+                        .sorted(eldsteFoerst().reversed())
+                        .collect(toList())
+                ).map(populerTilDato)
+                        .map(a -> a.stream()
+                                .sorted(eldsteFoerst())
```

Comment:
Dette ble litt rotete - men tror det kan være lurt å tilby metoder på objektet for å returnere listen i ulike rekkefølger.